build/docker/clusterrole.yaml

```diff
@@ -0,0 +1,10 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: kmesh
+  labels:
+    app: kmesh
+rules:
+- apiGroups: [""]
+  resources: ["pods","services","namespaces"]
+  verbs: ["get", "update", "patch"]
```

Comment:
miss watch？

Reply:
I think we donot need `watch` now

Reply:
I added patch, because we may need to patch label on the pod.